pool/database/seed/seed_user.ml

```diff
@@ -19,23 +19,36 @@ let answer_custom_fields fields contact =
   let open Custom_field in
   let open Public in
   let select_random options =
-    Random.int (List.length options) |> CCList.nth options |> Answer.create
+    Random.int (List.length options) |> CCList.nth options
   in
   CCList.filter_map
     (fun field ->
       match (field : Public.t) with
       | Select (public, options, _) ->
-        Public.Select (public, options, select_random options |> CCOption.pure)
+        Public.Select
+          ( public
+          , options
+          , select_random options
+            |> CCOption.pure
+            |> Answer.create
+            |> CCOption.pure )
         |> CCOption.pure
       | MultiSelect (public, options, _) ->
         Public.MultiSelect
-          (public, options, select_random options |> CCList.pure)
+          ( public
+          , options
+          , select_random options
+            |> CCList.pure
+            |> CCOption.pure
+            |> Answer.create
+            |> CCOption.pure )
         |> CCOption.pure
       | Boolean _ | Number _ | Text _ -> None)
     fields
   |> CCList.map (fun field ->
```

Comment:
Isn't here an issue, that `PartialUpdate` is a module and function?

in case variant exists
```
  |> CCList.map CCFun.(Custom_field.(PartialUpdate.custom %> PartialUpdate (update, contact, Pool_context.Contact contact))))
```

Reply:
No, one is a module, the other one is an event. There are no variants, used:

```
|> CCList.map (fun field ->
       let open Custom_field in
       PartialUpdate
         (PartialUpdate.Custom field, contact, Pool_context.Contact contact))
```